internal/db/seeds/dev_small.sql

```diff
@@ -4,18 +4,18 @@
 --
 BEGIN;
 
+-- Seed some account numbers, artificial or stage environment
 INSERT INTO accounts(id, account_number, org_id)
-VALUES (1, '13', '000013'),
-       (2, '15', '000015'),
-       (3, NULL, '000042'),
-       (4, NULL, '000077'),
-       (5, '6089719', '000016')
+VALUES (1, '13', '000013'), -- non-existing account
+       (2, NULL, '000042'), -- non-existing account
+       (3, '6395343', '13446659') -- stage account
 ON CONFLICT DO NOTHING;
 
+-- Seed some pubkeys, feel free to add your own key and associate it with your account
 INSERT INTO pubkeys(id, account_id, name, body, fingerprint)
-VALUES (1, 1, 'lzap-ed25519-2021',
-        'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEhnn80ZywmjeBFFOGm+cm+5HUwm62qTVnjKlOdYFLHN lzap',
-        'SHA256:gL/y6MvNmJ8jDXtsL/oMmK8jUuIefN39BBuvYw/Rndk')
+VALUES
+(1, 3, 'lzap-ed25519-2021',
+ 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEhnn80ZywmjeBFFOGm+cm+5HUwm62qTVnjKlOdYFLHN lzap', 'SHA256:gL/y6MvNmJ8jDXtsL/oMmK8jUuIefN39BBuvYw/Rndk')
 ON CONFLICT DO NOTHING;
 
```

Comment:
I cleaned the seed script a bit removing those arbitrary accounts and only keeping 13/13 and NULL/42. Then I moved my own pubkey to my stage account, feel free to add your own stage numbers and pubkeys if you want them after seed.